flink-core/src/main/java/org/apache/flink/api/connector/source/lib/util/IteratorSourceReader.java

```diff
@@ -19,6 +19,7 @@
 package org.apache.flink.api.connector.source.lib.util;
 
 import org.apache.flink.api.connector.source.ReaderOutput;
+import org.apache.flink.api.connector.source.SourceOutput;
```

Comment:
I don't get this change. Could you expand commit message or add a test case?

Reply:
I added an explanation in the commit message.